lib/zcaps.js

```diff
@@ -184,16 +193,25 @@ exports.id = async () => `urn:zcap:${await EdvClient.generateId()}`;
 exports.delegate = async ({zcap, signer, capabilityChain}) => {
   capabilityChain =
     Array.isArray(capabilityChain) ? capabilityChain : [zcap.parentCapability];
-  // attach capability delegation proof
-  return sign(zcap, {
-    // TODO: map `signer.type` to signature suite
-    suite: new Ed25519Signature2018({
+  let suite;
+  if(signer.type === 'Ed25519VerificationKey2018') {
+    suite = new Ed25519Signature2018({
       signer,
       verificationMethod: signer.id
-    }),
+    });
+  } else if(signer.type === 'Ed25519VerificationKey2020') {
+    suite = new Ed25519Signature2020({
+      signer,
+      verificationMethod: signer.id
+    });
+  }
+  // attach capability delegation proof
+  return sign(zcap, {
+    suite,
     purpose: new CapabilityDelegation({
       capabilityChain
     }),
-    compactProof: false
+    compactProof: false,
```

Comment:
Is `compactProof` an option anymore? I think we got rid of it in 8.x:

https://github.com/digitalbazaar/jsonld-signatures/blob/fe51dcfcd4567b364a99626cdbe2aa136aae0ab7/CHANGELOG.md#removed

Reply:
Removed 6644d2e